### DISCUSSION TITLE
Bind Trix to window

### DISCUSSION THREAD

**Discussion by sinnbeck** [filamentphp/filament#7876] [Ideas]
Any chance we can get Trix bound to the window? I am trying a add mentions to an editor, but that means I need to get access to the Trix instance.

https://github.com/basecamp/trix#inserting-a-content-attachment

If I just try to import it again, everything breaks, as it is loaded twice.

**Comment by danharrin** [comment]
You should copy the component and modify it instead of trying to extend it

**Reply by sinnbeck** [reply]
I was just hoping to keep all other updates to the component. If I copy it, any fixes etc to RichEditor will be lost in my version. I am simply adding a small script to handle the mention logic. This is just for my own project, not a plugin

**Reply by danharrin** [reply]
Yeah, but updates could also break your changes. It's just safer to copy instead of modify. We don't have many plans to change the rich editor.
